### ISSUE TITLE
[1.19.2] Crash on startup - java.lang.IllegalAccessError

### ISSUE THREAD

**Issue by DeleteMetaInf** [AbdElAziz333/Canary#14]
**Launcher console:** https://pastebin.com/raw/VYKWbkh8
**Log:** https://pastebin.com/raw/3rAaP7zV

Nothing unusual appears in the log, and no crash report is generated, but the errors in the launcher’s console show that the crash was caused by Canary, and disabling Canary lets the game boot as expected.

### Versions
* **Minecraft:** 1.19.2 (latest)
* **Forge:** 43.1.27 (latest)
* **Canary:** 0.0.2 (latest)

**Comment by AbdElAziz333** [comment]
are you sure you don't forgot any mod dependency or updating forge for the required version in your modpack?
because java.lang.IllegalAccessError happens only when you forget any mod dependency or forge version is bad.

**Comment by OcrSu** [comment]
> are you sure you don't forgot any mod dependency or updating forge for the required version in your modpack? because java.lang.IllegalAccessError happens only when you forget any mod dependency or forge version is bad.

This crash also appears in my modpack, but it is 1.18.2 Forge.
[debug.log](https://github.com/AbdElAziz333/Canary/files/9655734/debug.log)

**Comment by OcrSu** [comment]
I'm trying to find mods that conflict with canary. But I have been busy with exams recently. More useful information may not be available in the short term.

**Comment by OcrSu** [comment]
fixed in 1.18.2

> are you sure you don't forgot any mod dependency or updating forge for the required version in your modpack? because java.lang.IllegalAccessError happens only when you forget any mod dependency or forge version is bad.

**Comment by OcrSu** [comment]
fixed in 1.18.2

> are you sure you don't forgot any mod dependency or updating forge for the required version in your modpack? because java.lang.IllegalAccessError happens only when you forget any mod dependency or forge version is bad.

**Comment by HellishINC** [comment]
I'm also experiencing this issue on 1.19.2 with Canary build 0.0.3. Disabling it allows MC to run normally.

I'm convinced it's a mod conflicting as radium also crashes.

```
at TRANSFORMER/patchouli@1.19.2-76/vazkii.patchouli.client.handler.BookCrashHandler.appendToCrashReport(BookCrashHandler.java:23)
```

Frustratingly , patchouli is not at fault as removing it shows a different mod there (and it keeps going with different mods). I'll try to remove each mod listed in its place and see if that fixes it.

No Canary: https://mclo.gs/k9imGFP

Canary: https://mclo.gs/ZHAGItt

Radium: https://mclo.gs/mJeEVKI

**Comment by Pineappleguy03** [comment]
@AbdElAziz333 VERY COOL NOTE TO CHECK OUT HERE.

**Comment by AbdElAziz333** [comment]
This is incompatbility with debugify, for now you can fix this issue by adding `mixin.entity.collisions.unpushable_cramming`=false
 to the canary.properties file in the config folder.

**Comment by AbdElAziz333** [comment]
If you got java.lang.IllegalAccessError error that means you are missing a mod dependency or update forge version to your game, this is weird error and it may take time to fix.

**Comment by Chaos02** [comment]
> If you got java.lang.IllegalAccessError error that means you are missing a mod dependency or update forge version to your game, this is weird error and it may take time to fix.

You're right, I AM getting a missing mandatory. However it says missing dependency: minecraft, but not form which mod!
[latest.log](https://pastebin.com/ZMhK8VD5)
debug.log also doesn't contain it but it's too large for pastebin

**Comment by AbdElAziz333** [comment]
This issue is completely fixed in Canary 0.0.10 for MC 1.18.2, and Canary 0.1.1 for MC 1.19.2.
